utility/LocoNetMessageBuffer.cpp

```diff
@@ -72,24 +72,23 @@ LocoNetMessageBufferClass::LocoNetMessageBufferClass()
 
 lnMsg *LocoNetMessageBufferClass::getMsg(void)
 {
+
 	if( index < 2)
 		return NULL;
-		
+
 	if(!expLen)
-			// If it's a fixed length packet, compute the length from the OPC_Code, else get the length from the byte 1  
+			// If it's a fixed length packet, compute the length from the OPC_Code, else get the length from the byte 1
 		expLen = ( ( buffer[0] & (uint8_t)0x60 ) == (uint8_t)0x60 ) ? buffer[1] : ( ( buffer[0] & (uint8_t)0x60 ) >> (uint8_t)4 ) + (uint8_t)2 ;
-	
+
 	if(expLen == index)
 	{
 		if(checkSum == buffer[index-1])
 		{
 			stats.rxPackets++;
 			return (lnMsg*)buffer;
 		}
-		else
-			stats.rxErrors++;

```

Comment:
How are rxErrors counted if this line is removed?

Reply:
Hmm good point, missed that one